R/target_market_share.R

```diff
@@ -21,9 +21,9 @@
 #' outputting production, weighted by relative loan-size. Set to `FALSE` to
 #' output the unweighted production values.
 #'
-#' @return A tibble with the summarized columns `metric`, `production` and
-#' `technology_share`. If `by_company = TRUE`, the output will also have the
-#' column `name_ald`.
+#' @return A tibble including the summarized columns `metric`, `production` and
+#'   `technology_share`. If `by_company = TRUE`, the output will also have the
+#'   column `name_ald`.
```

Comment:
I think the original paragraph is unclear. I interpret it as meaning that the input and output have the same names except that the output has the new columns `metric`, `production`, `technology_share`, and `name_ald` (if `by_company = TRUE`) -- which is not true:

``` r
devtools::load_all()
#> ℹ Loading r2dii.analysis

input <- fake_matched()
output <- target_market_share(
  input,
  ald = fake_ald(),
  scenario = fake_scenario(),
  region_isos = region_isos_stable
)

setdiff(names(input), names(output))
#>  [1] "id_loan"                         "loan_size_outstanding"          
#>  [3] "loan_size_outstanding_currency"  "loan_size_credit_limit"         
#>  [5] "loan_size_credit_limit_currency" "id_2dii"                        
#>  [7] "level"                           "score"                          
#>  [9] "name_ald"                        "sector_ald"
```

So I added "including" to be more vague and more likely correct. If you can think of a better wording -- one that better reflects the returned value -- then please go ahead and edit.

Reply:
I think this looks pretty good to me!